src/App.vue

```diff
@@ -59,16 +59,16 @@ export default {
   data: () => ({
     componentLinks: [
       {
-        to: '/ImageContainer',
-        label: 'Image Container',
+        to: '/FgImageContainer',
+        label: 'Fg Image Container',
       },
       {
-        to: '/Card',
-        label: 'Card',
+        to: '/FgCard',
+        label: 'FgCard',
       },
       {
-        to: '/Table',
-        label: 'Table',
+        to: '/FgTable',
+        label: 'FgTable',
```

Comment:
```suggestion
        label: 'Table',
```

Reply:
I prefer not having the prefix here.

Reply:
@kamerat would it cause a conflict without a prefix since <table> is reserved for standard tables already ?

Reply:
@zashton, I think we're just looking to remove the prefix from the side menu labels and the routes.  The actual component should still be `<fg-table/>`

Reply:
@logan-laughery oh ok, sorry got confused here. then this change should be all good.